Homework/ElectricalAppliances/Services/HouseService.cs

```diff
@@ -0,0 +1,137 @@
+﻿using AutoMapper;
+using ElectricalAppliances.Models;
+using ElectricalAppliances.Enums;
+using ElectricalAppliances.Entities;
+using ElectricalAppliances.Repositories.Abstractions;
+using ElectricalAppliances.Extensions;
+
+namespace ElectricalAppliances.Services
+{
+	public class HouseService : IHouseService
+	{
+		private List<ElectronicDevice> availableDevices = new();
+        private House myHouse;
+
+
+        private readonly IRepository<ElectronicDeviceEntity> _eDevicesRepository;
+		private readonly ILoggerService _logger;
+		private readonly IMapper _mapper;
+
+		public HouseService(ILoggerService logger, IRepository<ElectronicDeviceEntity> eDevicesRepository, IMapper mapper)
+		{
+			_eDevicesRepository = eDevicesRepository;
+			_logger = logger;
+			_mapper = mapper;
+            availableDevices = _mapper.Map<List<ElectronicDevice>>(_eDevicesRepository.GetAll());
+            myHouse = new()
+            {
+                HouseDevices = new(),
+                ElectricalOutlets = new ElectricalOutlet[]
+                {
+                    new ElectricalOutlet(),
+                    new ElectricalOutlet(),
+                    new ElectricalOutlet(),
+                    new ElectricalOutlet(),
+                    new ElectricalOutlet()
+                }
```

Comment:
For what we need this?

Reply:
According to the conditions, it was necessary to connect the device